src/app/controllers/encrypt-submissions.server.controller.js

```diff
@@ -215,10 +215,55 @@ exports.saveResponseToDb = function (req, res, next) {
     })
 }
 
+function getMetadataForSingleSubmission(req, res, submissionId) {
```

Comment:
avoid passing `req` and `res` into a function, because the server will crash if a response is attempted twice.